app/services/hyrax/workflow/changes_required_notification.rb

```diff
@@ -8,15 +12,24 @@ def subject
         end
 
         def message
-          "#{title} (#{work_id}) requires additional changes before approval.\n\n '#{comment}'"
+          "#{title} (#{link_to work_id, document_path}) requires additional changes before approval.\n\n '#{comment}'"
         end
 
       private
 
+        def solr_document
```

Comment:
@jcoyne I gave that a try originally and the entity seems to be some kind of sipity item, not a solr document.  The original code was loading the document from fedora.